workflow/rocoto/workflow_xml.py

```diff
@@ -118,6 +118,12 @@ def _get_cycledefs_cycled(self):
         sdate = sdate + to_timedelta(interval)
         strings.append(f'\t<cycledef group="gdas">{sdate.strftime("%Y%m%d%H%M")} {edate.strftime("%Y%m%d%H%M")} {interval}</cycledef>')
 
+        if self._app_config.do_jedilandda:
+            sdate_land = sdate.replace(hour=18, minute=0, second=0)
+            edate_land = edate
+            if edate >= sdate:
+                strings.append(f'\t<cycledef group="gdas_land_prep">{sdate_land.strftime("%Y%m%d%H%M")} {edate_land.strftime("%Y%m%d%H%M")} 24</cycledef>')
```

Comment:
Workflow Q for you. So landanlinit, run, final will run every cycle, the prep only 18z (for the IMS). Where does the different dependency depending on cycle come into play in the XML generation?

Reply:
I haven't reviewed the `landanlinit` and `landanlrun` scripts yet on how they use the observation data created from `preplandobs`.  Once I see what that is, I may be able to add a dependency of `preplandobs` on the 18z cycle of `landanlinit` only.
I don't have an answer until I can review the actual use of this observation.

Reply:
got it, okay we will hold for now

Reply:
I think we resolved this.